app/models.py

```diff
@@ -399,8 +399,8 @@ class Service(db.Model, Versioned):
     go_live_user_id = db.Column(UUID(as_uuid=True), db.ForeignKey('users.id'), nullable=True)
     go_live_user = db.relationship('User', foreign_keys=[go_live_user_id])
     go_live_at = db.Column(db.DateTime, nullable=True)
-    sending_domain = db.Column(db.String(255), nullable=True, unique=False)
-    smtp_user = db.Column(db.String(255), nullable=True, unique=False)
+    sending_domain = db.Column(db.Text(255), nullable=True, unique=False)
+    smtp_user = db.Column(db.Text(255), nullable=True, unique=False)
```

Comment:
Why this change?

Reply:
"String" was changed to "Text" in the model to match the `sending_domain` and `smtp_user` column type we have in the database